internal/pkg/utils/testfs/fs.go

```diff
@@ -39,24 +35,3 @@ func NewMemoryFsFrom(localDir string) filesystem.Fs {
 	}
 	return memoryFs
 }
-
-func MinimalProjectFs(t *testing.T) filesystem.Fs {
```

Comment:
Presunul som tuto funkciu do `fixtures` pkg, kedze to pouziva subory, ktore su tam ulozene.